src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -156,6 +152,10 @@ private void clearPemFile() {
     pemFileClearButton.setVisible(false);
   }
 
+  private boolean isInvalidStartDate(LocalDate date) {
```

Comment:
I'd suggest renaming this to `isAfterEndDate(LocalDate date)`

Reply:
Done